snaps-cli.js

```diff
@@ -277,7 +280,7 @@ function applyConfig() {
 
     if (pkg.web3Wallet) {
       const { bundle } = pkg.web3Wallet;
-      if (bundle?.local) {
```

Comment:
We'd have to bump the Node version or add transpiling to use optional chaining with `.js`.